### ISSUE TITLE
[1.9] rpos

### ISSUE THREAD

**Issue by davelab6** [commontype-standard/commontype#20]
There's a thread on the mpeg mailing list with details

**Comment by simoncozens** [comment]
Yeah, I've just done a test implementation of this and now I'm not sure that I'm right. ;-) Because positioning values aren't "carried over" between glyphs, I don't think it matters what order you run the positioning rules in.

**Comment by davelab6** [comment]
So would this be a more fundamental update to the whole shaping model, and therefore a "CommonType-Next" issue?
